From 60b9d78ae7347bcedaed4ef028db9a179dad9575 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 31 Mar 2015 15:47:05 +0100
Subject: [PATCH 2/4] Remove warning in non-fatal absence of
 /etc/ssl/openssl.cnf

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 apps/openssl.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/apps/openssl.c b/apps/openssl.c
index 6873145..1263b47 100644
--- a/apps/openssl.c
+++ b/apps/openssl.c
@@ -337,7 +337,6 @@ int main(int Argc, char *ARGV[])
     if (i == 0) {
         if (ERR_GET_REASON(ERR_peek_last_error())
             == CONF_R_NO_SUCH_FILE) {
-            BIO_printf(bio_err, "WARNING: can't open config file: %s\n", p);
             ERR_clear_error();
             NCONF_free(config);
             config = NULL;
-- 
2.11.1

