From f6dcfdd21aa7850182fe845d0f9d4ec44999951f Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 22 Sep 2016 11:44:28 -0700
Subject: [PATCH 5/6] Make openssl stateless.

---
 crypto/cryptlib.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/crypto/cryptlib.h b/crypto/cryptlib.h
index fba180a..e339539 100644
--- a/crypto/cryptlib.h
+++ b/crypto/cryptlib.h
@@ -81,8 +81,8 @@ extern "C" {
 
 # ifndef OPENSSL_SYS_VMS
 #  define X509_CERT_AREA          OPENSSLDIR
-#  define X509_CERT_DIR           OPENSSLDIR "/certs"
-#  define X509_CERT_FILE          OPENSSLDIR "/cert.pem"
+#  define X509_CERT_DIR           "/var/cache/ca-certs/extracted/openssl"
+#  define X509_CERT_FILE          X509_CERT_DIR "/ca-bundle.trust.pem"
 #  define X509_PRIVATE_DIR        OPENSSLDIR "/private"
 # else
 #  define X509_CERT_AREA          "SSLROOT:[000000]"
-- 
2.10.0

