--- openssl-1.0.2h/crypto/x86_64cpuid.pl~	2016-05-03 13:44:42.000000000 +0000
+++ openssl-1.0.2h/crypto/x86_64cpuid.pl	2016-12-27 15:48:28.231137895 +0000
@@ -53,6 +53,7 @@
 .size	OPENSSL_rdtsc,.-OPENSSL_rdtsc
 
 .globl	OPENSSL_ia32_cpuid
+.hidden OPENSSL_ia32_cpuid
 .type	OPENSSL_ia32_cpuid,\@function,1
 .align	16
 OPENSSL_ia32_cpuid:
--- openssl-1.0.2h/crypto/x86cpuid.pl~	2016-05-03 13:44:42.000000000 +0000
+++ openssl-1.0.2h/crypto/x86cpuid.pl	2016-12-27 15:49:18.733489382 +0000
@@ -383,5 +383,6 @@
 
 &hidden("OPENSSL_cpuid_setup");
 &hidden("OPENSSL_ia32cap_P");
+&hidden("OPENSSL_ia32_cpuid");
 
 &asm_finish();
